app/screens/home/channel_list/categories_list/categories/body/category_body.tsx

```diff
@@ -25,25 +24,20 @@ type Props = {
 
 const extractKey = (item: ChannelModel) => item.id;
 
-const CategoryBody = ({sortedChannels, category, hiddenChannelIds, limit, onChannelSwitch, unreadChannels}: Props) => {
+const CategoryBody = ({sortedChannels, category, limit, onChannelSwitch, unreadChannels}: Props) => {
     const serverUrl = useServerUrl();
     const ids = useMemo(() => {
-        let filteredChannels = sortedChannels;
-
-        // Remove all closed gm/dms
-        if (hiddenChannelIds.size) {
-            filteredChannels = sortedChannels.filter((item) => item && !hiddenChannelIds.has(item.id));
-        }
+        const filteredChannels = sortedChannels;
 
         if (category.type === DMS_CATEGORY && limit > 0) {
             return filteredChannels.slice(0, limit);
         }
         return filteredChannels;
-    }, [category.type, limit, hiddenChannelIds, sortedChannels.length]);
+    }, [category.type, limit, sortedChannels]);
```

Comment:
@enahum I encountered that when a new post is created, the category-list component is not re-rendering. if we put into the array a number then the component is not re-rendering if the existing amount of channels does not change.

Reply:
I see, then I guess there is no point in memoizing this, as collapsing and expanding the category will re-calculate this as well anyway.

Any thoughts on the matter?